openfisca_aotearoa/variables/acts/income_tax/family_scheme/family_tax_credit.py

```diff
@@ -12,7 +12,12 @@ class family_scheme__qualifies_for_family_tax_credit(Variable):
     reference = "http://www.legislation.govt.nz/act/public/2007/0097/latest/DLM1518484.html"
 
     def formula(persons, period, parameters):
-        return persons("family_scheme__base_qualifies", period)
+        family_income = persons.family.sum(persons.family.members(
+            "family_scheme__assessable_income", period.this_year))
+        threshold = parameters(
+            period).entitlements.income_tax.family_scheme.family_tax_credit.full_year_abatement_threshold
+        income_under_threshold = family_income < threshold
```

Comment:
We should create another variable to do the calculation involving the abatement threshold, I believe the calculation is somewhat more complicated and will look at it now.

Reply:
This is the formula: http://www.legislation.govt.nz/act/public/2007/0097/latest/DLM1518512.html

Reply:
I agree we can add the calculation later, but smart start need only the boolean of qualifying, and not having the income test is blocking their work.

Reply:
can we have a variable something like   ...below_threshold?

Reply:
We've done more research and there isn't a threshold for WFF. you can earn millions if you have enough dependents.
Instead we are now implementing the actual calculation, and we'll also make a boolean variable that tells you if they qualify for more than $0.00